src/panels.cpp

```diff
@@ -1205,6 +1205,11 @@ static void draw_char( avatar &u, const catacurses::window &w )
     }
 
     mvwprintz( w, 2, 8, focus_color( u.focus_pool ), "%s", u.focus_pool );
+    if( u.focus_pool < u.calc_focus_equilibrium() ) {
+        mvwprintz( w, 2, 11, c_light_green, "↥" );
```

Comment:
could we use simple "+/-" or "</>" ? I'm not sure if this special character can show on all terminals since it is not in ASCII table.